ui/src/app/pages/access/compliance-training-module-card-title.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+
+import { AccessModule, Profile } from 'generated/fetch';
+
+import { AoU } from 'app/components/text-wrappers';
+import { profileApi } from 'app/services/swagger-fetch-clients';
+import { AccessTierShortNames } from 'app/utils/access-tiers';
+import {
+  getAccessModuleStatusByName,
+  isCompliant,
+  isExpiringOrExpired,
+} from 'app/utils/access-utils';
+
+export interface ComplianceTrainingModuleCardProps {
+  tier: AccessTierShortNames;
+  profile: Profile;
+}
+
+export const ComplianceTrainingModuleCardTitle = ({
+  tier,
+  profile,
+}: ComplianceTrainingModuleCardProps) => {
+  const [useAbsorb, setUseAbsorb] = React.useState(false);
+  useEffect(() => {
+    profileApi().useAbsorb().then(setUseAbsorb);
+  }, []);
```

Comment:
This is inefficient and is called many times when the page is loaded, due to several renders of this component. It's difficult to change the code to only call this once.

I think this is OK for now for the following reasons:
- This call will go away once we transfer users off of Moodle entirely
- This endpoint is fairly lightweight
- This page isn't visited often
- This call will go away if we solve deep links